mailbag/controller.py

```diff
@@ -97,6 +97,12 @@ def generate_mailbag(self):
         attachments_dir = os.path.join(str(mailbag_dir), "data", "attachments")
         error_dir = os.path.join(parent_dir, str(self.args.mailbag_name) + "_errors")
 
+        #Getting a list of all the files
+        file_list=[]
+        for root, dirs, files in os.walk(parent_dir):
+            for file in files:
+                file_list.append(file)
+
```

Comment:
I don't think you need this block here. You should be able to use the `os.walk()` on line 227 for this.